android/pytorch_android/style_transfer_cyclegan/src/main/java/ai/djl/examples/styletransfer/StyleTransferActivity.kt

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ * with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+ * OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package ai.djl.examples.styletransfer
+
+import ai.djl.android.core.BitmapImageFactory
+import android.Manifest
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.graphics.Bitmap
+import android.os.Bundle
+import android.provider.MediaStore
+import android.view.View
+import android.widget.*
+import android.widget.AdapterView.OnItemSelectedListener
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.app.ActivityCompat
+import androidx.core.content.ContextCompat
+
+class StyleTransferActivity : AppCompatActivity() {
+
+    private var styler: Styler? = null
+    private var imageView: ImageView? = null
+
+    private companion object {
+        const val CAMERA_PERMISSION_CODE = 1
+        const val REQUEST_IMAGE_CAPTURE = 2
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+
+        imageView = findViewById(R.id.style)
+
+        Thread {
+            styler = Styler()
+        }.start()
+
+        setupArtists()
+
+        val capture = findViewById<ImageButton>(R.id.capture)
+        capture.setOnClickListener {
```

Comment:
We should not enable camera button if model is not loaded. But we can fix this issue in the following PR.